packages/storybook/stories/wc-helpers.jsx

```diff
@@ -54,6 +70,11 @@ export const propStructure = comp => {
     };
     return propObj;
   }, {});
+
+  const events = getEventObj(comp.events);
+  const listeners = getEventObj(comp.listeners);
+
+  return { ...props, ...events, ...listeners };
```

Comment:
Instead of manually listing the events for each component, can we use this data?

Reply:
That's a great idea. Updated.

Reply:
Ah, now that I'm taking a closer look at this I see that we're putting the events in with the props. I'm not sure that's very clear, but others may feel differently.

Reply:
I think you are correct that this could create confusion and that it should be easily identifiable as something separate like in the Telephone example. 

I want to call out that `docs-json` doesn't include description details for listeners like it does for events.
```js
export interface JsonDocsListener {
  event: string;
  target?: string;
  capture: boolean;
  passive: boolean;
}
```

```js
export interface JsonDocsEvent {
  event: string;
  bubbles: boolean;
  cancelable: boolean;
  composed: boolean;
  docs: string;
  docsTags: JsonDocsTag[];
  deprecation?: string;
  detail: string;
}
```

I had to manually override it through Storybook argTypes to add descriptions. I think we might be able to add a JSDoc comment in each component for listeners that gets added to a component's `docsTags`, and use that to return a modified docs using https://stenciljs.com/docs/docs-custom. I haven't tried it yet, but I think this is what we want?